test/assert.js

```diff
@@ -0,0 +1,114 @@
+var node_assert = require('assert')
+
+function isAssertionError (error) {
+  return error.name.startsWith('AssertionError')
+}
+
+/**
+ * Picks the location of the assertion from the stacktrace if available.
+ *
+ * @param {Error} error
+ */
+function locateAssertion (error) {
+  return error && error.stack
+    && error.stack.split('\n').find(
```

Comment:
```suggestion
  return error && error.stack &&
    error.stack.split('\n').find(
```

Reply:
We could also put the whole return expression in parens.

In general, I would like to avoid potential confusion due to the way JavaScript handles return expressions (which I think I do not understand 100%).

I think the Prettier tool would follow my suggestion, with the break after binary operator. (Prettier does put the break before ternary operators, which took me a while to get used to. They seem to be considering changing the split for binary operators in a future release, but not so easy to get sufficient level of consensus.)

Reply:
I think this change is just about formatting and from that perspective it's just a tiny part of it. 
Let's introduce prettier in some later PR and fix all those at once.
If you think it helps just apply the change, I gave you edit rights to the branch when I created the PR.